src/main/java/org/example/model/Game.java

```diff
@@ -0,0 +1,15 @@
+package org.example.model;
+
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
+public class Game {
+    private int id;
+    private String name;
+    private String release_date;
```

Comment:
release_date -> releaseDate

Reply:
why String?

Reply:
> why String?

My bad, just have corrected it here and in User class, I also refactored the code which are related from these fields.